packages/system/0.3.0/docs/README.md

```diff
@@ -0,0 +1,618 @@
+# System Integration
+
+The System module allows you to monitor your servers. Because the System module
+always applies to the local server, the `hosts` config option is not needed.
+
+The default metricsets are `cpu`, `load`, `memory`, `network`, `process`, and
+`process_summary`.
+
+Note that certain metricsets may access `/proc` to gather process information,
+and the resulting `ptrace_may_access()` call by the kernel to check for
+permissions can be blocked by
+https://gitlab.com/apparmor/apparmor/wikis/TechnicalDoc_Proc_and_ptrace[AppArmor
+and other LSM software], even though the System module doesn't use `ptrace`
+directly.
+
+## Compatibility
+
+The System metricsets collect different kinds of metric data, which may require dedicated permissions
+to be fetched and which may vary across operating systems.
+
+## Metrics
```

Comment:
Seems like we only have `## Metrics` session here but not for logs?

Reply:
All of that is auto-generated by...something? Not sure.